formsfeeder.client/src/main/java/formsfeeder/client/support/Builder.java

```diff
@@ -28,6 +28,10 @@
 
 	public Builder basicAuthentication(String username, String password);
 
+	public Map<String, Supplier<Object>> getQueryParams();
+
+	public Builder addQueryParam(String name, Supplier<Object> value);
+

```

Comment:
Per my main comment. This becomes `addQueryParam(String name, List<Supplier<String>> value)`.

I would also add two default methods to make the client code a little more succinct.
```java
	public default Builder addQueryParam(String name, Supplier<String> value) {
		addQueryParam(name, Arrays.asList(value));	// List.of() would be better, but we're in Java 8 land here
		return this;
	}

	public default Builder addQueryParam(String name, String value) {
		this.addQueryParam(name, ()->value);
		return this;
	}
```